raft.go

```diff
@@ -250,6 +314,24 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 		return nil, err
 	}
 
+	// Scan through the log for any configuration change entries
+	snapshotIndex, _ := r.getLastSnapshot()
+	for index := snapshotIndex + 1; index <= lastLog.Index; index++ {
+		var entry Log
+		if err := r.logs.GetLog(index, &entry); err != nil {
+			r.logger.Printf("[ERR] raft: Failed to get log at %d: %v", index, err)
+			panic(err)
+		}
+		// TODO: support LogAddPeer, LogRemovePeer too
+		if entry.Type == LogConfiguration {
+			r.configurations.committed = r.configurations.latest
```

Comment:
Since we are reading out of the log, shouldn't these always flow to both slots in the `configurations` struct? This looks like committed will be one step behind.